DXRMissions/DeusEx/Classes/DXRMissionsParis.uc

```diff
@@ -153,9 +153,8 @@ function MissionTimer()
 
     switch(dxr.localURL) {
     case "10_PARIS_METRO":
-        if(dxr.flags.settings.goals > 0)
-            UpdateGoalWithRandoInfo('MeetJaime', "Jaime could be anywhere in Paris.");
-        UpdateGoalWithRandoInfo('GetCrack', "The zyme can be anywhere.");// not actually depedent on goals rando
+        UpdateGoalWithRandoInfo('MeetJaime', "Jaime could be anywhere in Paris.");
+        UpdateGoalWithRandoInfo('GetCrack', "The zyme can be anywhere.",True);// not actually depedent on goals rando
```

Comment:
this timer doesn't run if goals/starts rando is disabled

```
    if(dxr.flags.settings.startinglocations <= 0 && dxr.flags.settings.goals <= 0)
        return;

    SetTimer(1, true);
```

also this should probably be:
```suggestion
        if(dxr.flags.settings.swapitems > 0) {
            UpdateGoalWithRandoInfo('GetCrack', "The zyme can be anywhere.",True);// not actually depedent on goals rando
        }
```

Reply:
The swapitems check is implemented in 3e3b2e1 but the timer issue remains if goal rando is disabled